controllers/labels.go

```diff
@@ -17,6 +27,7 @@ func ServiceLabels(name string) map[string]string {
 	return map[string]string{
 		"clusterName": name,
 		"app":         "infinispan-pod",
+		"version":     version(),
```

Comment:
I think we should be consistent here with the label names and use "infinispan_version".

Reply:
label name should conform k8s recommended rules https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ and could be like
`infinispan.org/version`